bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuntimeCondition.java

```diff
@@ -26,7 +27,8 @@
     private Set<Connection> connections;
 
     public RuntimeCondition(Condition condition) {
-        super(condition.getId(), condition.getTypeUID(), condition.getConfiguration(), condition.getInputs());
+        super(condition.getId(), condition.getTypeUID(),
+                new Configuration(condition.getConfiguration().getProperties()), condition.getInputs());
```

Comment:
> We don't have a setters because these are public immutable objects

Err - they all extend `Module`, which has public setters (incl. the mentioned `setConfiguration`. But anyhow, `setConfiguration` could even be private and could be called in the constructor of `Module` - that was my question (2) above.